scripts/linters/pre_commit_linter.py

```diff
@@ -453,6 +454,17 @@ def _print_errors_stacktrace(errors_stacktrace):
         ' above errors gets fixed')
 
 
+def _get_space_separated_linter_name(linter):
```

Comment:
The param should be linter_class_name*

Reply:
Actually it is not the linter class name but the linter object from which we are extracting class name with `type(linter).__name__`

Reply:
Sorry for not being clear, I mean to pass the className as a param to this function instead of the whole object. (Basically I mean provide least data to this function.)